backend/src/migrations/1543523539612_init.js

```diff
@@ -0,0 +1,98 @@
+exports.shorthands = undefined;
+
+exports.up = (pgm) => {
+    pgm.createTable('users', {
+        email: { type: 'varchar(254)', primaryKey: true},
```

Comment:
Did you mean to set it to 256 chars? (here and below)

Reply:
I copied this numbers from my previous django project. They are just to start of with something, and discuss and change them accordingly. I personally think that even 128 would be more than enough for `email`, but we can do 256 If you think its better.